tests/e2e/matmul/BUILD.bazel

```diff
@@ -62,6 +62,31 @@ py_binary(
     "large",
 ]]
 
+# LLVMCPU, non-data-tiling, no microkernels, ArmSME
+[iree_generated_trace_runner_test(
+    name = "e2e_matmul_arm_sme_nondt_%s_%s" % (dtype, size),
```

Comment:
I think it is okay to add a new entry for the test suite. But can we remove the original one? E.g., can we remove line 50 in this file?

Reply:
The tests added here are just for `SME`, line 50 adds tests for `SVE`. I put the `SME` tests in their own section as (for now), SME is a bit more of a special case.

Reply:
oops, I misread it. Sorry about that and good catch!